options/posix/include/arpa/inet.h

```diff
@@ -9,11 +9,6 @@
 extern "C" {
 #endif
 
-uint32_t htonl(uint32_t);
```

Comment:
we shouldn't remove these from this header, musl has these functions defined in both so I think we should match that behaviour.